data/json/items/armor.json

```
@@ -7036,7 +7036,7 @@
     "name": "pearl collar",
     "description": "A collar made from round and lustrous pearls, its former owner must be a wealthy individual.",
     "weight": 30,
-    "volume": 0,
+    "volume": "0 L",
```

```suggestion
    "volume": "300 ml",
```
Going off of [this pearl collar](https://www.amazon.com/Freshwater-Cultured-Necklace-Stunning-Bracelet/dp/B00CW8SZE6/ref=sr_1_6?keywords=pearl+necklace&qid=1566235203&s=gateway&sr=8-6) of these dimensions 7.6 x 7.5 x 1.6 inches, then multiplying by 0.2.